src/main/java/org/folio/service/manager/export/strategy/InstanceExportStrategyImpl.java

```diff
@@ -109,7 +116,17 @@ private void generateRecordsOnTheFly(ExportPayload exportPayload, List<String> i
     exportPayload.setExportedRecordsNumber(srsLoadResult.getUnderlyingMarcRecords().size() - failedSrsRecords + mappedMarcRecords.size() - failedRecordsCount);
     exportPayload.setFailedRecordsNumber(identifiers.size() - exportPayload.getExportedRecordsNumber());
     if (exportPayload.isLast()) {
-      exportService.postExport(fileExportDefinition, params.getTenantId());
+      try {
+        exportService.postExport(fileExportDefinition, params.getTenantId());
+      } catch (ServiceException exc) {
+        jobExecutionService.getById(exportPayload.getJobExecutionId(), params.getTenantId()).onSuccess(res -> {
+          Optional<JsonObject> optionalUser = usersClient.getById(fileExportDefinition.getMetadata().getCreatedByUserId(),
```

Comment:
Checking that user is present could be useful here.

Reply:
Alright, fixed.